### ISSUE TITLE
This gem needs love. ASAP. It is not even compatible with Guard anymore.

### ISSUE THREAD

**Issue by costa** [suranyami/guard-delayed#11]
@suranyami Any ideas? Will you call for and choose a new maintainer? Thanks.

**Comment by substantial-philipbjorge** [comment]
@suranyami -- I'm running into the same issue as @costa ...
My understanding of pessimistic versioning is that the dependency now is saying guard-delayed requires guard > 0 and < 1...

So for those of us on Guard 2, we can't install this.

I'll take a stab at verifying this.

**Comment by costa** [comment]
Hey, and now it is not compatible with delayed_job 4, what gives?)

**Comment by substantial-philipbjorge** [comment]
@costa -- what's it complaining about? The gemspec says delayed_job >= 0...

**Comment by substantial-philipbjorge** [comment]
Either way, it fixed it for us :stuck_out_tongue_winking_eye: , so thanks

**Comment by costa** [comment]
```
    guard-delayed (~> 0.3.1) ruby depends on
      delayed_job (~> 3) ruby depends on
        activesupport (~> 3.0) ruby

    rails (~> 4.1.6) ruby depends on
      activesupport (4.1.6)
```

**Comment by costa** [comment]
ping

**Comment by suranyami** [comment]
Check version 0.3.2. Should have fixed the above issues @costa

**Comment by costa** [comment]
@suranyami Thanks!

**Comment by suranyami** [comment]
Just released version 0.3.5:

* Improved documentation on the optional parameters that can be passed in.
* Executable command now defaults to `bin/delayed_job` in-line with recent Rails versions.

I'm gonna close this issue, but if anyone else is still having problems, please lodge a new issue with details.
